crates/typst-pdf/src/page.rs

```diff
@@ -717,9 +717,12 @@ fn write_text(ctx: &mut PageContext, pos: Point, text: &TextItem) {
     let fill_transform = ctx.state.transforms(Size::zero(), pos);
     ctx.set_fill(&text.fill, true, fill_transform);
     if let Some(stroke) = &text.stroke {
-        ctx.set_stroke(stroke, true, fill_transform);
-        ctx.content
-            .set_text_rendering_mode(pdf_writer::types::TextRenderingMode::FillStroke);
+        if stroke.thickness.to_f32() > 0.0 {
```

Comment:
special handling for 0pt stroke. per https://github.com/typst/typst/issues/3579#issuecomment-1985285908 . I don't know if i'm doing right